monkey/monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -67,6 +68,13 @@ class ReportPageComponent extends AuthComponent {
             ransomwareReport: res
           });
         });
+      this.authFetch('/api/telemetry?telem_category=file_encryption')
+        .then(res => res.json())
```

Comment:
Use `IslandHttpClient`. It removes the need for redundant `.then(res => res.json())` and allows you to access status of the response if needed.